ocaml/idl/sdk-README.html

```diff
@@ -1,110 +0,0 @@
-<html>
```

Comment:
I believe these files used to be needed for the content on the old community page, but they're out of date and I don't think they're currently published anywhere. In any case I'll have another look to ensure the changes are not breaking anything.